src/inmanta/server/protocol.py

```diff
@@ -208,24 +209,34 @@ async def stop(self) -> None:
 
         order = list(reversed(self._get_slice_sequence()))
 
-        exceptions: dict[str, Exception] = {}
+        pre_stop_exceptions: dict[str, Exception] = {}
+        stop_exceptions: dict[str, Exception] = {}
+
         for endpoint in order:
             try:
                 LOGGER.debug("Pre Stopping %s", endpoint.name)
                 await endpoint.prestop()
             except Exception as e:
-                exceptions[endpoint.name] = e
+                pre_stop_exceptions[endpoint.name] = e
+
+        if pre_stop_exceptions:
+            LOGGER.debug("The following errors occurred during pre-stopping: %s", str(pre_stop_exceptions))
```

Comment:
Isn't it weird to 1) log a debug message and 2) raise an exception. Both of them will appear in the log in the end. I feel like we should remove 1)